### PR TITLE
MF-662 - Change menu style

### LINKED ISSUE #662: Improve the menu
- Highlight selected item
- Menu should be a nav-bar and not a button group. Try to add boostrap menu here

### PR DESCRIPTION
**Pull request by blokovi**
### What does this do?
Change menu style:
- update menu from bootstrap button group to bootstrap nav
- highlight selected item

### Which issue(s) does this PR fix/relate to?
Resolves #662

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: ui/src/Main.elm [hunk 1]
```diff
@@ -310,11 +310,13 @@ view model =
...
-                    , ButtonGroup.linkButton [ Button.primary, Button.onClick Channels, buttonAttrs ] [ i [ class "fas fa-broadcast-tower" ] [], text " Channels" ]
-                    , ButtonGroup.linkButton [ Button.primary, Button.onClick Connection, buttonAttrs ] [ i [ class "fas fa-plug" ] [], text " Connection" ]
-                    , ButtonGroup.linkButton [ Button.primary, Button.onClick Messages, buttonAttrs ] [ i [ class "far fa-paper-plane" ] [], text " Messages" ]
+                    [ ul [ class "nav-pills flex-column nav" ]
+                        [ menuItem "Dashboard" Version "fas fa-chart-bar" (model.view == "dashboard")
+                        , menuItem "Things" Things "fas fa-sitemap" (model.view == "things")
+                        , menuItem "Channels" Channels "fas fa-broadcast-tower" (model.view == "channels")
+                        , menuItem "Connection" Connection "fas fa-plug" (model.view == "connection")
+                        , menuItem "Messages" Messages "far fa-paper-plane" (model.view == "messages")
+                        ]
```
Comment (darkodraskovic) [inline ui/src/Main.elm:319]: Ok for `ul`. Please replace `as fa-chart-bar` (magic string) and similar by some constant pulled from some module.

**Review by darkodraskovic — DISMISSED** [review]
LGTM

**Review by darkodraskovic — DISMISSED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**